bin/ci-publish.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -ex
+
+npmPublish() {
+  npm publish
+}
+
+currentVersion() {
+  grep version <package.json | sed 's/[^0-9.]*\([0-9.]*\).*/\1/'
+}
+
+currentProject() {
+  npm ls --depth=-1 | head -1 | cut -f 1 -d " " | sed 's/@.*//'
+}
+
+checkVersion() {
+  local gitTag
+  gitTag=$(git describe --tags)
+
+  if [ "$(currentVersion)" != "$gitTag" ]; then
```

Comment:
Oh, I didn't realize that `git describe --tags` contains a hash of the commit. If we're comparing commit hashes, then this will work, but the `currentVersion` doesn't have a hash in it, so this will never match.